hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/state/DispatchingEvmFrameState.java

```diff
@@ -110,7 +110,7 @@ public class DispatchingEvmFrameState implements EvmFrameState {
     // The only exception is that the function selector for `redirectForScheduleTxn` (0x5c3889ca)
     // has been pre substituted before the ADDRESS_BYTECODE_PATTERN.
     private static final String SCHEDULE_CALL_REDIRECT_CONTRACT_BINARY =
-            "6080604052348015600f57600080fd5b50600061016a9050775c3889cafefefefefefefefefefefefefefefefefefefefe600052366000602037600080366018016008845af43d806000803e8160008114605857816000f35b816000fdfea2646970667358221220d8378feed472ba49a0005514ef7087017f707b45fb9bf56bb81bb93ff19a238b64736f6c634300080b0033";
+            "6080604052348015600f57600080fd5b50600061016b9050775c3889cafefefefefefefefefefefefefefefefefefefefe600052366000602037600080366018016008845af43d806000803e8160008114605857816000f35b816000fdfea2646970667358221220d8378feed472ba49a0005514ef7087017f707b45fb9bf56bb81bb93ff19a238b64736f6c634300080b0033";
```

Comment:
I detect _one letter_ in that hex string changed.  Don't even know if it is the right change.  I suggest again:  This is the 4 byte hex _selector_ that needs to be substituted into these three otherwise identical 147 byte hex strings.  Can't we have code that takes the one true proxy binary and injects that selector into it producing these three static strings?

Reply:
well there's actually 2 changes.  I see what I can do.